lib/get-macro-options/index.js

```diff
@@ -13,12 +13,21 @@ function getMacroOptionsJson (componentName) {
   return JSON.parse(fs.readFileSync(optionsFilePath, 'utf8'))
 }
 
-function addSlugs (option) {
-  // camelCase into kebab-case
-  option.slug = (option.name).replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase()
+function addWordBreaksAndSlugs (option) {
   if (option.params) {
-    option.params = option.params.map(addSlugs)
+    option.params = option.params
+      .map(addWordBreaksAndSlugs)
   }
+
+  // Add URL-safe slugs for hrefs and IDs
+  option.slug = option.name
+    .replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase()
+
+  // Add suggested word breaks before capital letters to allow
+  // browsers to break long option names in sensible places
+  option.name = option.name
+    .replace(/([A-Z])/g, '<wbr>$1')
```

Comment:
I've added "work break opportunities" before each capital letter

Reads much better by stopping `textareaDescriptionText` etc from breaking mid-word

## Before

<img width="622" alt="Option name  Description  is broken mis-word to  Descripti" src="https://github.com/alphagov/govuk-design-system/assets/415517/e9de1a73-ddb5-463a-8276-d6e338fa73e9">

## After

<img width="623" alt="Word break opportunity prevents  Description  from breaking mid-word" src="https://github.com/alphagov/govuk-design-system/assets/415517/9f307965-2fb6-4a33-8451-1ee99ea1fd26">

Reply:
Moved to https://github.com/alphagov/govuk-design-system/pull/2927